packages/patrol_finders/lib/src/custom_finders/patrol_tester.dart

```diff
@@ -243,18 +287,26 @@ class PatrolTester {
     SettlePolicy? settlePolicy,
     Duration? visibleTimeout,
     Duration? settleTimeout,
+    bool enablePatrolLog = true,
   }) {
-    return TestAsyncUtils.guard(() async {
-      final resolvedFinder = await waitUntilVisible(
-        finder,
-        timeout: visibleTimeout,
-      );
-      await tester.tap(resolvedFinder.first);
-      await _performPump(
-        settlePolicy: settlePolicy,
-        settleTimeout: settleTimeout,
-      );
-    });
+    return wrapWithPatrolLog(
+      action: 'tap',
+      finder: finder,
+      color: '33m',
```

Comment:
What color is this? I think this should also be wrapped with something more descriptive